CHANGELOG.md

```diff
@@ -22,6 +22,9 @@ Please document your changes in this format:
 
 ## [Unreleased]
 
+### Fixed
+- ics export for individual activities was fixed in the mobile app @amengsk [#2400]
```

Comment:
- At the end, it should be the id of this PR and not of the issue -> change to `[#2406]`
- text suggestion: `add explicit host in ics url`
- I also suggest to change PR title accordingly: `fix: add explicit host in ics url`

Sorry, I am a really a nitpicker when it comes to stuff like this 😆